papers/bradley_dice/55_bradley_dice.rst

```diff
@@ -0,0 +1,563 @@
+Bundling is what allows the submission script to contain multiple jobs executing the same operation.
+By storing information about the generated bundles during submission, **signac-flow** prevents accidental resubmission just as in the unbundled case.
+While the example above does not use either groups or aggregation, bundles works seamlessly with both.
+
+Synced Collections: Backend-agnostic, persistent, mutable data structures
+-------------------------------------------------------------------------
+
+Motivation
+~~~~~~~~~~
+
+At its core, **signac** is a tool for organizing and working with data on the filesystem, presenting a Pythonic interface for tasks like creating directories and modifying files.
+In particular, **signac** makes modifying the JSON files used to store the state points and documents as easy as working with Python dictionaries.
+Despite heavily optimizing these APIs, in seeking to scale **signac** to ever-larger data spaces we quickly realized that the most significant barrier intrinsic to our model was the overhead of parsing and modifying large numbers of text files.
+Unfortunately, the usage of JSON files in this manner was deeply embedded in our data model, making switching to a more performant backend without breaking APIs or severely complicating our data model a daunting task.
+
+While attempting to overcome this hurdle, we identified a core problem that our current API for modifying JSON files was solving in a limited manner: namely, the creation of a dictionary-like interface to an underlying resource.
+Numerous other well-known Python packages (h5py and Zarr immediately come to mind) also do this in order to make working with complex resources feel as natural as possible to Python users.
+Most such packages implement this layer directly for their particular use case, but the nature of the problem suggested to us the possibility of developing a more generic representation of this interface.
+Indeed, the Python standard library's ``collections.abc`` module is designed exactly for the purpose of making it easy to define objects that "look like" standard Python objects while having completely customizable behavior under the hood.
+As such, we saw an opportunity to specialize this pattern for a specific use case: the transparent synchronization of a Python object with an underlying resource.
+
+The **synced collections** framework represents the culmination of our efforts in this direction, providing a generic framework within the interfaces of any abstract data type can be mapped to arbitrary underlying synchronization protocols.
+In **signac** this framework allows us to hide the details of a particular file storage medium (like JSON) behind a dictionary-like interface, but it can just as easily be used for tasks like creating a set-like interface to an underlying extension type or wrapping a directory manager in a list-like interface.
```

Comment:
```suggestion
In **signac**, this framework allows us to hide the details of a particular file storage medium (like JSON) behind a dictionary-like interface, but it can just as easily be used for tasks such as creating a set-like interface to an underlying extension type or wrapping a directory manager in a list-like interface.
```